scripts/setup.sh

```diff
@@ -2,13 +2,10 @@
 export MYDIR=`dirname $0`
 
 # Universe top level directory
-export UNIVERSE=$MYDIR/..
+export UNIVERSE=$MYDIR/../
 
 # CheckerFramework Inference
-export CFI="$JSR308"/checker-framework-inference
-
-# Generic Type Inference Solver
-export SOLVER="$JSR308"/generic-type-inference-solver
+export CFI="$MYDIR"/../../checker-framework-inference
 
 # Dependencies
-export CLASSPATH=$UNIVERSE/bin:$CHECKERFRAMEWORK/dataflow/build:$CHECKERFRAMEWORK/javacutil/build:$CHECKERFRAMEWORK/stubparser/build:$CHECKERFRAMEWORK/framework/build:$CHECKERFRAMEWORK/checker/build:$SOLVER/bin:$CHECKERFRAMEWORK/framework/tests/junit-4.12.jar:$CHECKERFRAMEWORK/framework/tests/hamcrest-core-1.3.jar:$CFI/bin:$CFI/dist/org.ow2.sat4j.core-2.3.4.jar:$JSR308/jsr308-langtools/build/classes:$CLASSPATH
+export CLASSPATH=$CFI/bin:$CFI/dist/org.ow2.sat4j.core-2.3.4.jar:$CFI/dist/commons-logging-1.2.jar:$CFI/dist/log4j-1.2.16.jar:./build/libs/universe.jar:$CFI/dist/checker-framework-inference.jar:$CLASSPATH
```

Comment:
Does `$CFI/bin` contain compiled `.class` files? I think we should remove this and, if at all, add the directories where gradle puts its output.